src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectNftSettingsPage/ProjectNftSettingsPage.tsx

```diff
@@ -0,0 +1,276 @@
+import { t, Trans } from '@lingui/macro'
+import { useForm } from 'antd/lib/form/Form'
+import { Button, Form } from 'antd'
+import { JuiceInput } from 'components/inputs/JuiceTextInput'
+import { ProjectMetadataContext } from 'contexts/shared/ProjectMetadataContext'
+import { V2V3ProjectContext } from 'contexts/v2v3/Project/V2V3ProjectContext'
+import { useSetThemeTx } from 'hooks/v2v3/transactor/SetThemeTx'
+import { CSSProperties, useContext, useState } from 'react'
+import styles from './Capsules.module.css'
```

Comment:
does `module` have any meaning for us? could we just call it `css` and it Just Works?

Reply:
Ah nice, TIL that Next supports css modules out of the box. Did not know this!

LGTM.